src/ds/d400/d400-options.h

```diff
@@ -76,12 +80,17 @@ namespace librealsense
         hw_monitor& _hwm;
         sensor_base* _sensor;
         std::weak_ptr<limits_option> _exposure_limit_toggle;
+        bool _ae_gain_limits_new_opcode;
```

Comment:
No need to mention gain in AE option, rename `_new_opcode`.
Same goes for other occurences of this name.